### DISCUSSION TITLE
Info about Sibionics sensor connection

### DISCUSSION THREAD

**Discussion by AlessioLei94** [j-kaltes/Juggluco#181] [Q&A]
Hi, we are trying to integrate out system with Sibionics glucose sensor and we came across your app that from what we see in the README should be able to connect and communicate with such sensors.

Doing some tests with a sensor and the Sibionics app we see that there's the need to send specific commands to the sensor in order to successfully connect to it and (we assume) to setup/start the glucose readings.

Could you confirm that the app is able to connect and collect data from Sibionics GS1 sensors?
Is the app actually sending any command to the sensor as we assume there's the need to? And if yes could you point out where that's done in you app so we can have a look at it?

Thanks in advance,
Alessio

**Comment by j-kaltes** [comment]
You can use Sibionics sensors with Juggluco. Other apps can receive the glucose values from Juggluco via [glucose broadcasts](https://www.juggluco.nl/Juggluco/glucosebroadcast.html) and the [web server](https://www.juggluco.nl/Jugglucohelp/Nightscouthelp.html) in Juggluco.
I don't help people with putting the code of Juggluco in their own app.

**Reply by AlessioLei94** [reply]
Sorry that's my bad i didn't specify it but we are not trying to build an app.

We have an IoT device that acts as gateway to connect to different sensors for health monitoring and collect readings that are then sent over to a dashboard.

We were asked to integrate such sensor with out system and it's always nice to have help from someone who figured things out already, but of course it's up to you to decide if you want to give these info or not.

Alessio
